--- pylint-2.9.6/pylint.egg-info/requires.txt~	2021-07-28 18:51:37.000000000 +0000
+++ pylint-2.9.6/pylint.egg-info/requires.txt	2021-08-16 15:58:03.719720160 +0000
@@ -1,4 +1,4 @@
-astroid<2.7,>=2.6.5
+astroid>=2.6.5
 isort<6,>=4.2.5
 mccabe<0.7,>=0.6
 toml>=0.7.1
--- pylint-2.9.6/setup.cfg~	2021-07-28 18:51:38.000000000 +0000
+++ pylint-2.9.6/setup.cfg	2021-08-16 15:58:26.210719847 +0000
@@ -42,7 +42,7 @@
 [options]
 packages = find:
 install_requires = 
-	astroid>=2.6.5,<2.7 # (You should also upgrade requirements_test_min.txt)
+	astroid>=2.6.5 # (You should also upgrade requirements_test_min.txt)
 	isort>=4.2.5,<6
 	mccabe>=0.6,<0.7
 	toml>=0.7.1
